modules/ROOT/partials/configuration/advcode.adoc

```diff
@@ -0,0 +1,24 @@
+[[advcode_inline]]
+== `+advcode_inline+`
+
+include::partial$misc/admon-requires-6.3v.adoc[]
+
+As part of the {productname} 6.3 release, the {pluginname} plugin includes a new option, `advcode_inline`, that allows users to open the Advanced Code Editor within {productname}’s existing editor space instead of been displayed in a seperate dialog box.
```

Comment:
```suggestion
As part of the {productname} 6.3 release, the {pluginname} plugin includes a new option, `advcode_inline`, that allows users to open the Advanced Code Editor within {productname}’s existing editor space instead of being displayed in a separate dialog box.
```